packages/portal-proto/src/features/repositoryApp/FilesTable.tsx

```diff
@@ -483,7 +493,7 @@ const FilesTables: React.FC = () => {
           search={{
             enabled: true,
             tooltip:
-              "e.g. HCM-CSHL-0062-C18.json, 4b5f5ba0-3010-4449-99d4-7bd7a6d73422",
+              "e.g. HCM-CSHL-0062-C18.json, 4b5f5ba0-3010-4449-99d4-7bd7a6d73422, HCM-CSHL-0062-C18",
```

Comment:
Nitpick: I'm not sure how often this will change or how many places will use it. If we make this a constant and import it, we can easily update it in one place later if needed.

Reply:
so this specific text will be used in 2 files table. each search "tooltip" will have different text depending on what search is possible. not sure if 2 is sufficient to make a const out of it. 🤷‍♀️ any idea?

Reply:
Okay it makes sense if that's the case. We can leave as is.